doc/api/events.md

```diff
@@ -262,7 +262,8 @@ By default, a maximum of `10` listeners can be registered for any single
 event. This limit can be changed for individual `EventEmitter` instances
 using the [`emitter.setMaxListeners(n)`][] method. To change the default
 for *all* `EventEmitter` instances, the `EventEmitter.defaultMaxListeners`
-property can be used.
+property can be used. If trying to set it to a value which is not a positive
```

Comment:
Can I suggest rewording this as:

`If this value is not a positive number, a TypeError will be thrown.`